src/Wallabag/CoreBundle/Helper/Tools.php

```diff
@@ -118,4 +118,16 @@ public static function generateToken()
 
         return str_replace('+', '', $token);
     }
+
+    /**
+     * For a given text, we calculate reading time for an article.
+     *
+     * @param $text
+     *
+     * @return float
+     */
+    public static function getReadingTime($text)
+    {
+        return floor(str_word_count(strip_tags($text)) / 200);
```

Comment:
Could you just add a comment about the 200 number? Telling that it is referencing to _words per minute_